packages/cli-config/package.json

```
@@ -9,6 +9,7 @@
   "types": "build/index.d.ts",
   "dependencies": {
     "@react-native-community/cli-tools": "^10.0.0-alpha.0",
+    "chalk": "^5.1.2",
```

Let's use "^4.1.2" to align with other packages.
```suggestion
    "chalk": "^4.1.2",
```